src/components/ShareToken/ShareToken.js

```diff
@@ -37,22 +37,19 @@ const ShareToken = (props) => {
 
   return (
     <>
-      <p>Join an existing shopping list by entering a three word token.</p>
+      <h2>Welcome to your smart shopping list!</h2>
+      <p>Join an existing shopping list by entering a three word token:</p>
       <StyledForm id="shareToken" onSubmit={onSubmitHandler}>
         <label htmlFor="shareToken">Share token</label>
```

Comment:
Caitlin, I wonder if this doesn't need this label ^.

It looks like the MUI `TextField` has a label prop and that is also a label, so I'm getting a "double-label" error with WAVE.
![image](https://user-images.githubusercontent.com/10378331/120050219-3bb27f80-bfe2-11eb-9cdc-6c618cb17d13.png)

Reply:
Whoops, good call! Guess I went a little overboard trying to make sure I labeled everything 😅